src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Microsoft.Performance.Toolkit.PluginManager.Core.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="Discovery\" />
+    <Folder Include="Manager\" />
+    <Folder Include="Registry\" />
+    <Folder Include="Publishing\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EditorConfigFiles Remove="D:\playground\microsoft-performance-toolkit-sdk\src\PluginManager\Microsoft.Performance.Toolkit.PluginManager.Core\.editorconfig" />
```

Comment:
Did you mean to leave this and the one below in?

Reply:
This got auto populated when I added an .editorconfig file to a project (right click add new EditorConfig). Not sure why it added a `Remove` here. I removed this anyways. Makes sense to add it to the sln in another PR.